docs/userstories.rst

```diff
@@ -16,14 +16,16 @@ Names:
 * Sylvester (John Rambo)
 * Jean Reno (Leon der Profi, Mathilda)
 
-In this user story, the output of the workflow is a scientific paper describing the research results.
-An essential component of this work is the development of research software in line with the task area `BETTY <https://nfdi4ing.de/archetypes/betty/>`_.
```

Comment:
I removed the reference to `Betty` here because I thought this would rather be confusing for readers that haven't yet read about the structure of NFDI4Ing.